src/docs/reference/teams.md

```diff
@@ -2,16 +2,14 @@
 title: Teams
 ---
 
-Teams are how organizations are represented within Railway. Teams with a valid credit card attached are subscribed to the Team plan. Projects within the Team plan get generous limits on service caps such as 32 GB of RAM and 32 vCPU cores.
+Teams are how organizations are represented within Railway. Teams can be created via the Hobby, Pro, or Enterprise plans.
```

Comment:
>Teams can be created via the Hobby, Pro, or Enterprise plans.

Teams can only be created for Pro and Enterprise, right?

Reply:
fixed in 9f1b7b0f0b4811e245fedee26b205b8df461f515